### DISCUSSION TITLE
Emails not the same

### DISCUSSION THREAD

**Discussion by trevorschadt** [community/community#6939] [Copilot Conversations] [answered]
My Github email is not the same as the email I'm using for my IntelliJ license. Thus, while I received an email (to my Github email address) to join the program, I am unable to install the plugin in IntelliJ. Is there any way to reconcile these? Thanks!

**Comment by jansorg** [comment, marked as answer]
The GitHub Copilot plugin isn't tied to your IntelliJ license or the email used for this in any way. 
Please follow https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md#installing to install it and to log in to GitHub.

**Reply by trevorschadt** [reply]
Here's what I get when I try to install it from IntelliJ:
![image](https://user-images.githubusercontent.com/5910459/139490389-fd155419-5425-4295-828c-c8df65081d29.png)

**Reply by jansorg** [reply]
@trevorschadt That's weird and seems related to the installation of your IDE. Which IDE and version are you using? Are you able to install other plugins?

**Reply by trevorschadt** [reply]
IntelliJ IDEA 2021.2.3 (Ultimate Edition)
Build #IU-212.5457.46, built on October 12, 2021

Yes, I am able to install other plugins.

**Reply by jansorg** [reply]
The file is fetched from JetBrains' servers, there's nothing we can do about the missing URL.
You could try restarting your IDE to force a refresh. Alternatively, you could try downloading the latest release from https://plugins.jetbrains.com/plugin/17718-github-copilot/versions/stable/143458 and drag&drop it onto your IDE to install it (or choose "install from disk" in the plugin manager settings).

**Reply by trevorschadt** [reply]
Downloading the plugin from the marketplace did the trick. Thanks!
